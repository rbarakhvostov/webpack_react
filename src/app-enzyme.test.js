import React from 'react';

import { mount } from 'enzyme';

import toJson from 'enzyme-to-json';

import App from './app.jsx';

describe('<App />', () => {
  const wrapper = mount(<App />);

  it('App renders without crashing', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('App renders without crashing (check h1)', () => {
    expect(wrapper.find('h1').text()).toEqual('COUNTER');
  });
});
