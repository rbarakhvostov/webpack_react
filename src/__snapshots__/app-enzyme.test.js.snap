// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> App renders without crashing 1`] = `
<App>
  <div
    className="container"
  >
    <img
      alt="avatar"
      src="test-file-stub"
    />
    <h1>
      COUNTER
    </h1>
    <span>
      0
    </span>
    <div>
      <button
        onClick={[Function]}
      >
        -
      </button>
      <button
        onClick={[Function]}
      >
        +
      </button>
    </div>
  </div>
</App>
`;
